src/native/corehost/browserhost/libBrowserHost.footer.js

```diff
@@ -54,6 +54,14 @@
 
                         // WASM-TODO: remove once globalization is loaded via ICU
                         ENV["DOTNET_SYSTEM_GLOBALIZATION_INVARIANT"] = "true";
+
+                        if (ENVIRONMENT_IS_NODE) {
+                            Module.preInit = [() => {
+                                FS.mkdir("/managed");
```

Comment:
What is copying assets to `managed` folder? 
WasmAppBuilder is using `_framework` as well https://github.com/dotnet/runtime/blob/4459d2006773767e7d23f139a20b27ec26f0be59/src/mono/browser/build/BrowserWasmApp.targets#L46

Reply:
One line below, I'm mounting the host directory `.` into the filesystem. It would be current directory of the nodejs process. `managed` folder is not ideal, I would much prefer to mount VFS root instead, but I don't know how yet.

`managed` name is hardcoded in PAL somewhere, so I just match that. But I guess we can change that.

`_framework` could happen to be subfolder of that or be current folder.

The intention is to get as close as possible to desktop probing
https://learn.microsoft.com/en-us/dotnet/core/dependency-loading/default-probing

Except we don't have ability to load multiple versions of runtime and we don't have dynamic libraries.